src/Infrastructure/BotSharp.Abstraction/Agents/Models/Agent.cs

```diff
@@ -46,6 +46,12 @@ public class Agent
     [JsonIgnore]
     public List<AgentTemplate> Templates { get; set; } = new();
 
+    /// <summary>
+    /// Links that can be filled into parent prompt
+    /// </summary>
+    [JsonIgnore]
+    public List<AgentLink> Links { get; set; } = new();
```

Comment:
It's not necessary to use a dedicated fields to store the linked templates.
They can be stored in `Templates` fields.